nano/node/bootstrap/bootstrap_lazy.cpp

```diff
@@ -489,7 +488,7 @@ void nano::bootstrap_attempt_wallet::request_pending (nano::unique_lock<nano::mu
 		auto account (wallet_accounts.front ());
 		wallet_accounts.pop_front ();
 		++pulling;
-		auto this_l (shared_from_this ());
```

Comment:
Is this correct? Why is bootstrap_attempt_lazy converting to bootstrap_attempt_wallet, it seems strange.

Reply:
It's in the bootstrap_lazy file but this is inside a bootstrap_attempt_wallet method: nano::bootstrap_attempt_wallet::request_pending